quipucords/scanner/openshift/api.py

```diff
@@ -291,3 +309,33 @@ def _init_cluster(self, cluster) -> OCPCluster:
             version=cluster["status"]["desired"]["version"],
         )
         return ocp_cluster
+
+    def _init_managed_cluster(self, cluster) -> dict:
+        managed_cluster_metrics = {}
+
+        key_map_labels = {
+            "vendor": "vendor",
+            "cloud": "cloud",
+            "version": "openshiftVersion",
+            "managed_cluster_id": "clusterID",
+        }
+
+        labels = cluster["metadata"]["labels"]
+        for metric_key, cluster_api_key in key_map_labels.items():
+            managed_cluster_metrics[metric_key] = labels[cluster_api_key]
+
+        conditions = cluster["status"]["conditions"]
+        is_cluster_available = any(
+            condition.get("reason") == "ManagedClusterAvailable"
+            for condition in conditions
+        )
+        managed_cluster_metrics["available"] = is_cluster_available
+
+        capacity = cluster["status"]["capacity"]
+        if capacity:
+            managed_cluster_metrics["core_worker"] = capacity.get("core_worker", "")
+            managed_cluster_metrics["socket_worker"] = capacity.get("socket_worker", "")
```

Comment:
None of the other facts collected on OCP scans can be absent and I don't see the need for an exception here.

```suggestion
        capacity = cluster["status"]["capacity"] or {}
        managed_cluster_metrics["core_worker"] = capacity.get("core_worker")
        managed_cluster_metrics["socket_worker"] = capacity.get("socket_worker")
```

Reply:
You have a point.  I will add this modification.